cli/flags.rs

```diff
@@ -190,6 +192,38 @@ Prettier dependencies on first run.
             .multiple(true)
             .required(true),
         ),
+    ).subcommand(
+      SubCommand::with_name("xeval")
+        .setting(AppSettings::DisableVersion)
+        .about("Eval a script on text segments from stdin")
+        .long_about(
+          "
+Eval a script on lines (or chunks split under delimiter) from stdin.
+
+Read from standard input and eval code on each whitespace-delimited
+string chunks.
+
+-I/--replvar optionally set variable name for input to be used in eval.
+Otherwise '$' will be used as default variable name.
+
+  cat list.txt | deno xeval 'console.log($)'
+  cat list.txt | deno xeval -I 'val' 'console.log(val)'
+  cat list.txt | deno xeval -d ' ' 'console.log($)'
+  cat list.txt | deno xeval -d $'\\n' 'console.log($)'
```

Comment:
I think examples that people can copy, paste, and run out of the box would be better. In particular, if we use a file that usually exists. Say `/etc/passwd`. 

Here's one that prints all the users on the system:
```
cat /etc/passwd | ./target/debug/deno xeval "a = $.split(':'); if (a) console.log(a[0])"
```

Reply:
This isn't xeval, but normal eval. Here's an obtuse way of counting the bytes of a file:
```
cat README.md | ./target/debug/deno eval "b = await Deno.readAll(Deno.stdin); console.log(b.length)"
```

Reply:
Or something like
```
git branch | deno xeval "if ($.startsWith('*')) console.log($.split()[1])"
```
to find current branch?

Reply:
Yes that's good. I'd modify it to be:
```
git branch | deno xeval "if ($.startsWith('*')) console.log($.slice(2))"
```

Reply:
I also try to find an example to use the `-d` flag. A quite contrived example is to check if somthing is MIT licensed:

```
cat LICENSE | deno xeval -d " " "if ($ === 'MIT') console.log('MIT licensed')"
```

Reply:
SGTM